packages/vats/src/core/basic-behaviors.js

```diff
@@ -83,6 +89,166 @@ export const makeVatsFromBundles = async ({
 };
 harden(makeVatsFromBundles);
 
+/** @param {BootstrapPowers} powers */
+export const produceStartUpgradeable = async ({
+  consume: { zoe },
+  produce, // startUpgradeable
```

Comment:
what would it take to put this function into `BootstrapPowers` next to `produce` ?   The `{ consume: { startUpgradeable: startUpgradeableP } }… const startUpgradeable = await startUpgradeableP;` could be replaced with `{ startUpgradeable }`